jersey-sample-poc/src/main/java/com/mh/jerseysamplepoc/RestClient.java

```diff
@@ -0,0 +1,81 @@
+package com.mh.jerseysamplepoc;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class RestClient {
+    private Client apiClient;
+    private WebTarget webTarget;

```

Comment:
You can make webTarget and response local variable as there is no need to keep the value in class scope